docs/input_processing.md

```diff
@@ -1,25 +1,17 @@
-As of now two inputs are supported:
-- JSON Schema Draft 7
-- AsyncAPI version 2.0.0
+# Input process
 
-## Internal model representation
-
-![Class diagram](./assets/class_diagram.png)
-
-As seen on the class diagram the `InputProcessor` is our main point of entry for processing input data. 
-
-It uses the defined input processors (`AsyncAPIInputProcessor`, `JsonSchemaInputProcessor`, ...) by first calling `shouldProcess` function of each and if the function returns true it calls the `process` function. 
+The [InputProcessor](../src/processors/InputProcessor.ts) is our main point of entry for processing input data. It uses the defined input processors (`AsyncAPIInputProcessor`, `JsonSchemaInputProcessor`, ...) by first calling `shouldProcess` function for each input processor to figure out if the input data should be processed. Once a processor have been found it 
```

Comment:
`...Once a processor have been found it` what then? 😄

Reply:
thaaats a good question 😆